app/src/main/java/com/eljaviluki/grindrplus/Obfuscation.kt

```diff
@@ -139,15 +139,15 @@ object Obfuscation {
 
                 const val ChatBaseFragmentV2 = "$_chat.ChatBaseFragmentV2"
                 object ChatBaseFragmentV2_ {
-                    const val _canBeUnsent = "X1"
```

Comment:
change this to read
`const val _canBeUnsent = "Y1"`
and everything except the preventing of recording profile views will be working again.